packages/ui/src/pages/JobPage/JobPage.tsx

```diff
@@ -0,0 +1,74 @@
+import { useParams, useHistory, useLocation } from 'react-router-dom';
+import { AppJob, AppQueue, JobRetryStatus, Status } from '@bull-board/api/typings/app';
+import React, { useState } from 'react';
+import { Store } from '../../hooks/useStore';
+import s from '../QueuePage/QueuePage.module.css';
+import { JobCard } from '../../components/JobCard/JobCard';
+import { ArrowLeftIcon } from '../../components/Icons/ArrowLeft';
+import { Button } from '../../components/JobCard/Button/Button';
+import { useInterval } from '../../hooks/useInterval';
+
+export const JobPage = ({
+  actions,
+  queue,
+  selectedStatus,
+}: {
+  queue: AppQueue | null;
+  actions: Store['actions'];
+  selectedStatus: Store['selectedStatuses'];
+}) => {
+  const { search } = useLocation();
+  const history = useHistory();
+  const { name, jobId } = useParams<any>();
+  const [job, setJob] = useState<AppJob>();
+  const [status, setStatus] = useState<Status>(selectedStatus[queue?.name || '']);
+
+  useInterval(() => {
+    fetchJob();
+  }, 5000);
```

Comment:
> should use users config of interval time
idk what this means - if you can point me to an example I'm happy to make the change

>Do we pause the interval of the QueuePage?
no - I just let the polling continue because it keeps everything in sync and it's simple. traffic is not a real concern for an app like this imo. You probably wouldn't have 1000s of concurrent users.